packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
+   
+   ```bash
+   ./chainweb-node --print-config > default-config.yaml
+   ```
+
+   If you're running the node in a Docker container, you can create a configuration file for the node by running the following command:
+   
+   ```bash
+   docker run --publish 1789:1789 --publish 1848:1848 --entrypoint=chainweb/chainweb-node ghcr.io/kadena-io/chainweb-node/ubuntu:latest --print-config > default-config.yaml
+   ```
+   
+   This command creates the default configuration file in the host environment outside of the Docker container so that you can review configuration options and settings before starting the node in the container.
+
+   After you create a node configuration file from the default settings, you should determine whether you want to make any changes to the configuration of the node.
+   If you want to modify any of the default settings or add features that are disabled by default, you can:
+
+   - Edit settings in one or more configuration files.
+   - Use corresponding command-line options to control node operations.
+   
+   For more information about editing configuration settings in the configuration file, see [Edit the configuration settings](#edit-the-configuration-settings).
+   For information about using command-line options to control node operations, see the [chainweb-node]() command-line reference.
+
+## Edit the configuration settings
+
+After you create a node configuration file from the default settings, you can edit the configuration to suit your environment.
+For example, you can edit the configuration file to make the following types of changes:
+
+- Enable the backup API and specify the directory for backup files.
+- Connect the node to the Kadena test network instead of the Kadena main network.
+- Specify a directory for blockchain database files.
+- Ignore or connect to a subset of [bootstrap nodes]().
+
+To edit the node configuration:
+
+1. Open a terminal shell on a computer with access to the node configuration file.
+2. Copy the default configuration file to save as a backup.
+   
+   For example, copy the `default-config.yaml` to save a copy of it as the `modified-config.yaml` file:
+
+   ```bash
+   cp default-config.yaml modified-config.yaml
+   ```
+
+3. Open the configuration file in a text editor and edit the settings you want to change.
+   
+   For example, the following excerpt illustrates some common changes:
+
+   ```yaml
+   chainweb:
+     allowReadsInLocal: false
+     backup:
+       api:
+         configuration: {}
+         enabled: true
+       directory: /tmp/my-backups
+     chainwebVersion: testnet04
+     
+     fullHistoricPactState: false
+     
+     databaseDirectory: /usr/local/share
+   ```
+
+## Start the Chainweb node
+
+After you've made any changes needed in the node configuration file, you can start the node with the modified configuration file or by specifying the command-line options you want to set.
```

Comment:
Maybe we have a systemd unit file somewhere that people could use to start their node too, as a more production-oriented environment.